libs/types/src/election.ts

```diff
@@ -115,6 +115,7 @@ export interface Candidate {
   readonly name: string;
   readonly partyIds?: readonly PartyId[];
   readonly isWriteIn?: boolean;
+  readonly writeInPosition?: number;
```

Comment:
I agree we should reconsider the types, and I'm glad you looked at it this PR for that reason. Unless you're kidding and I don't get it, I will rename `writeInIndex` so that when we refactor it's clear these are referring to similar things :trollface:. Shall I make a ticket to reconsider and then refactor these types? I think that should be done but punted past m14. 

I was also confused how the `Candidate` and `WriteInCandidate` types are related, and how the default `writeInCandidate` is actually of type `Candidate`

Reply:
> Shall I make a ticket to reconsider and then refactor these types? I think that should be done but punted past m14.

Yes please 🙏